komodo-core-tests/src/test/resources/org/komodo/repository/test/test-local-repository-on-file-config.json

```diff
@@ -84,6 +77,9 @@
         "removeDerivedContentWithOriginal" : true,
         "threadPool" : "modeshape-workers",
         "sequencers" : {
+            "VDB Dynamic Sequencer" : {
+				"classname" : "org.teiid.modeshape.sequencer.vdb.VdbDynamicSequencer"
```

Comment:
This does not make sense. The KDdlSequencer encompasses all the sequencers required so there should be no reason to have to randomly include a sequencer in the config file just to ensure the vdb.cnd loads.